cprover_bindings/src/irep/goto_binary_serde.rs

```diff
@@ -199,6 +202,23 @@ impl IrepNumberingInv {
     }
 }
 
+#[cfg(not(test))]
+/// A numbering of [InternedString], [IrepId] and [Irep] based on their contents.
```

Comment:
Why is it that this is only done for the non-test scenario? Asking differently: why can we not use this for testing?